server.js

```diff
@@ -1,10 +1,39 @@
 require('dotenv').config();
+const jwt = require('express-jwt');
+const { expressJwtSecret } = require('jwks-rsa');
 const express = require('express');
 const cors = require('cors');
 const bodyParser = require('body-parser');
 const path = require('path');
 const { trackMediaClicked } = require('./src/tracker');
 
+// Auth0 Config
+const { AUTH0_AUDIENCE, AUTH0_DOMAIN } = process.env;
+const AUTH0_ISSUER = `https://${AUTH0_DOMAIN}/`;
+
+// Authentication middleware. Please see:
+// https://auth0.com/docs/quickstart/backend/nodejs
+// for implementation details
+const checkJwt = jwt({
+  // Retrieve the signing key from the server
+  secret: expressJwtSecret({
+    cache: true,
+    rateLimit: true,
+    jwksRequestsPerMinute: 5,
+    jwksUri: `${AUTH0_ISSUER}.well-known/jwks.json`,
+    handleSigningKeyError: (err, cb) => {
```

Comment:
Good job with this Abiodun, however, I think it would be better to write `error` instaed of `err` and `callback` instead of `cb` for the sake of readability

Reply:
Plus 1 to what @Tony-Ndichu said.

Reply:
Thanks, @TheoOkafor and @Tony-Ndichu. The feedback has been implemented. Can you confirm that?

Reply:
Good job @sulenchy